### DISCUSSION TITLE
How can I respond with DateTime in a aware timezone?

### DISCUSSION THREAD

**Discussion by phyyou** [fastapi/fastapi#6767] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Description

I am using the DateTime field, and my computer is in a virtual environment, so I adjusted the timezone to suit my time zone.
like this,

```python
...
def get_kst_now() -> datetime:
    return datetime.now(timezone("Asia/Seoul"))
```
```python
class ...(Base):
    ...

    create_at = Column(DateTime(timezone=True))
    update_at = Column(DateTime(timezone=True))

```

 But the response was UTC timezone. like this,
```json
{
  ...
  "id": 1,
  "create_at": "2021-04-02T16:19:38.903182+00:00",
  "update_at": "2021-04-02T16:17:25.604858+00:00"
  ...
}
```

In django, can setting up Timezone information at settings.py to `USE_TZ = True` and `Timzone=".."` then, automatically supported.
So how can I change UTC to my aware timezone?

### Environment

* OS: Linux
* FastAPI Version 0.54.1:

* Python version: 3.9.1
* Postgresql 1.4

**Comment by dstlny** [comment, marked as answer]
This doesn't look like a FastAPI issue, it looks more like an issue with whichever ORM that you are using? (I'm assuming SQLAlchemy, based on the model).

You'd need to look at the documentation for that ORM to find out how to replace the time zone, since SQLAlchemy (by default) uses UTC, which is what you're seeing here. Off the top of my head though, this is a "default" keyword which you can pass to the columns, so you might be able to override it using that :)

**Comment by coder3112** [comment]
I think, by ``response``, OP means pydantic response models. A possible link can be to pydantic docs [here](https://pydantic-docs.helpmanual.io/usage/types/#datetime-types) but definitely not FastAPI. To see how to define response schema, here are the [docs](https://fastapi.tiangolo.com/tutorial/response-model/)

**Comment by phyyou** [comment]
> This doesn't look like a FastAPI issue, it looks more like an issue with whichever ORM that you are using? (I'm assuming SQLAlchemy, based on the model).
> 
> You'd need to look at the documentation for that ORM to find out how to replace the time zone, since SQLAlchemy (by default) uses UTC, which is what you're seeing here. Off the top of my head though, this is a "default" keyword which you can pass to the columns, so you might be able to override it using that :)

Thank you for your answer. I'm gonna looking up the SQLAlchemy document in more detail.

**Comment by tiangolo** [comment]
Thanks for the help here everyone! 👏  🙇 

Thanks for reporting back and closing the issue @phyyou  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
